index.html

```diff
@@ -14,11 +14,60 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Proza+Libre:wght@400;600&display=swap" rel="stylesheet">
 </head>
 <body>
-  <header>
+  <header class="purple-header">
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h2 class="thin-font">Code Journal</h2>
+        </div>
+      </div>
+    </div>
   </header>
-  <main>
+  <main class="grey-background">
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1 class="font28pt">New Entry</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-half">
+          <div>
+            <img src="images/placeholder-image-square.jpg" alt="placeholder-image" class="placeholder-img">
+          </div>
+        </div>
+        <div data-view="entry-form" class="column-half">
```

Comment:
The `data-view` attribute should be applied to an element that holds _all_ of the content for this view, which would be the title, image preview, and form.